packages/bahmni-widgets/src/documentPrintButton/DocumentPrintButton.tsx

```
@@ -0,0 +1,92 @@
+            onChange={({ selectedItem }) => {
+              if (selectedItem) handlePrint(selectedItem);
+            }}
+            label=""
+            type="inline"
+            size={size ?? 'lg'}
+            titleText=""
+            selectedItem={null}
+          />
+        </div>
+      )}
+    </>
```

ESLint

[error] Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all.

(react/jsx-no-useless-fragment)